plot.py

```diff
@@ -663,16 +668,19 @@ def _set_default_limits(self):
                 
     def _check_file_type(self):
         '''Check file to see if the file is airborne or rhi.'''
-        if self.Vradar.value.scan_type != 'rhi':
-            pass
+        radar = self.Vradar.value
+        if radar.scan_type != 'rhi':
+            return
         else:
-            try:
-                (self.Vradar.value.metadata['platform_type'] == 'aircraft_tail') or \
-                (self.Vradar.value.metadata['platform_type'] == 'aircraft')
-                self.airborne = True
-            except:
+            if 'platform_type' in radar.metadata:
+                if (radar.metadata['platform_type'] == 'aircraft_tail' or
+                    radar.metadata['platform_type'] == 'aircraft'):
+                    self.airborne = True
+                else:
+                    self.rhi = True
+            else:
                 self.rhi = True
-            
+            #XXX this has only one effect: self._set_default_limits()
```

Comment:
Should we also delete the function `_set_fig_ax_rhi` as it is not being used any more?